src/app/rmmapi/messagelist.service.ts

```diff
@@ -218,6 +218,8 @@ export class MessageListService {
         if (hasChanges) {
             this.messagesInViewSubject.next(this.folderMessageLists[this.currentFolder]);
             this.refreshFolderList();
+            console.log('New/Moved messages, updating folder counts');
```

Comment:
I assume this is left out intentionally, to ease the debugging in near future?

Reply:
Yes that was my thought